cmd/_integration-tests/transport/http_test.go

```diff
@@ -400,6 +400,67 @@ func TestStrangeRPCName(t *testing.T) {
 	}
 }
 
+func TestHTTPErrorStatusCodeAndNilHeaders(t *testing.T) {
+	// See handlers/handlers.go for implementation
+	// Returns status code http.StatusTeapot
+	req, err := http.NewRequest("GET", httpAddr+"/status/code", strings.NewReader(""))
```

Comment:
Like I mentioned in the other PR, it's possible to make a GET request with a body, but it's not usual and it seems strange to do it in a test. Making these POST requests would be clearer.